### DISCUSSION TITLE
Google Play Install Referrer

### DISCUSSION THREAD

**Discussion by raresn** [airsdk/Adobe-Runtime-Support#1747] [Q&A] [answered]
hey guys, we are trying to better understand how we spend our money on ads, and we need to know from where the users install our app.
Right now the options are:
- link to google play from our website
- directly from google play
- clicking on an ad in google play

But we don't know how to track this. 
We read something about google play install referrer, but don't know how we can use this in air.
https://developer.android.com/google/play/installreferrer

Any clue about this @ajwfrost or @marchbold 

Thank you!

**Comment by marchbold** [comment, marked as answer]
We have a free extension available to access the install referrer:

https://github.com/distriqt/ANE-AndroidSupport/wiki/com.android.installreferrer
